src/NuGet.Core/NuGet.Commands/Utility/MSBuildProjectFrameworkUtility.cs

```diff
@@ -255,14 +284,23 @@ public static class MSBuildProjectFrameworkUtility
                     currentFrameworkString = "Silverlight,Version=v4.0,Profile=WindowsPhone71";
                     return NuGetFramework.Parse(currentFrameworkString);
                 }
-                if (isCppCli) // Don't use the platform moniker to CPP/CLI.
-                {
-                    platformMoniker = null;
-                }
+
                 NuGetFramework framework = NuGetFramework.ParseComponents(currentFrameworkString, platformMoniker);
 
                 if (isCppCli)
                 {
+                    if (!string.IsNullOrEmpty(windowsTargetPlatformMinVersion))
+                    {
+                        if (!Version.TryParse(windowsTargetPlatformMinVersion, out Version cppCliVersion))
+                        {
+                            throw new ArgumentException(string.Format(
+                                CultureInfo.CurrentCulture,
+                                "Invalid platform min version: {0}",
```

Comment:
Should we have a localizable string? In `NuGet.VisualStudio`, there's a similar one, `SemanticVersionStringInvalid`.